packages-proprietary/tokens/src/components/ams/breadcrumb.tokens.json

```diff
@@ -15,7 +15,11 @@
       },
       "link": {
         "color": { "value": "{ams.links.color}" },
+        "margin-block": { "value": "calc(var(--ams-space-xs) * -1)" },
+        "margin-inline": { "value": "calc(var(--ams-space-xs) * -1)" },
```

Comment:
This causes the hit areas to overlap when the Breadcrumb wraps. I don't think this change is really necessary, the hit area is already larger than 24x24, and there's space around the links to make them easier to hit.